golang/testsuite/networks_impl/test_network.go

```diff
@@ -87,30 +107,30 @@ func (network *TestNetwork) SetupDatastoreAndTwoApis() error {
 
 	network.datastoreClient = datastoreClient
 
-	personModifyingApiClient, err := network.addApiService()
+	personModifyingApiClient, err := network.addApiService(ctx, datastoreServiceContext.GetIPAddress())
 	if err != nil {
 		return stacktrace.Propagate(err, "An error occurred adding the person-modifying API client")
 	}
 	network.personModifyingApiClient = personModifyingApiClient
 
-	personRetrievingApiClient, err := network.addApiService()
+	personRetrievingApiClient, err := network.addApiService(ctx, datastoreServiceContext.GetIPAddress())
 	if err != nil {
 		return stacktrace.Propagate(err, "An error occurred adding the person-retrieving API client")
 	}
 	network.personRetrievingApiClient = personRetrievingApiClient
 
-	return nil
+	return returnErr
```

Comment:
Same comment from https://github.com/kurtosis-tech/kurtosis-cli/pull/61 about not having a changing return value